packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 50.3.0 - October 28, 2020
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- New force:apex:log:list command implementation ([PR #2644](https://github.com/forcedotcom/salesforcedx-vscode/pull/2644))
```

Comment:
@ekapner this command is no directly surfaced in the command palette, it is part of another command (Get logs). What do you think, should we leave it as is or reference where it's used ?

Reply:
@lcampos - I like the idea of providing more information.

Reply:
how about `New force:apex:log:list command implementation, used as part of 'SFDX: Get Apex Debug Logs ...'` ?

Reply:
That works, thanks